src/components/TutorialWindowComponent.js

```diff
@@ -0,0 +1,102 @@
+import {
+  EventHandlerService,
+  PDFLEvents,
+} from "../services/EventHandlerService";
+/**
+ * Component representing tutorial window component and it's element
+ *
+ * @property {Object} components object that holds DOM elements that represent tutorial popup component and it's own objects
+ * @property {HTMLElement} components.panelTutorial div that contains the tutorial window
+ * @property {HTMLElement} components.highlightTutorialWindow card with higlight tutorial gif and text
+ * @property {HTMLElement} components.graphTutorialWindow card with knowledge graph tutorial gif and text
+ * @property {HTMLElement} components.summaryTutorialWindow card with summary tutorial gif and text
+ */
+class TutorialWindowComponent {
+  components = {
+    panelTutorial: document.querySelector("#tutorial"),
+    highlightTutorialWindow: document.querySelector(
+      "#highlight-tutorial-window"
+    ),
+    graphTutorialWindow: document.querySelector("#graph-tutorial-window"),
+    summaryTutorialWindow: document.querySelector("#summary-tutorial-window"),
+  };
+
+  /**
+   * Creates the handler service for managing mouse events on the questions marks elements
+   * @constructor
+   */
+  constructor() {
+    this.#registerEvents();
+  }
+
+  /**
+   * Adds event listeners to tutorial window component and it's elements.
```

Comment:
By reading code it looks like only "Adds event listeners to tutorial window component"